static/js/apply-for-jobs.js

```diff
@@ -0,0 +1,213 @@
+    });
+    datalistDisciplineElement.innerHTML = disciplineList;
+  };
+
+  async function getDataFromGreenhouseApi(category, term) {
+    if (term && (term != "")) {
+      const response = await fetch(`https://boards-api.greenhouse.io/v1/boards/Canonical/education/${category}?term=${term}`);
+      if (response.ok) {
+        const data = await response.json();
+        return data.items;
+      } else {
+        console.log("HTTP-Error: " + response.status);
+      }
+    } else {
+      const response = await fetch(`https://boards-api.greenhouse.io/v1/boards/Canonical/education/${category}`);
+      if (response.ok) {
+        const data = await response.json();
+        return data.items;
+      } else {
+        console.log("HTTP-Error: " + response.status);
+      }
+    }
+  }
+
+  function addEducationInput(n) {
+    const educationInput = document.createElement("div");
+    educationInput.setAttribute("data-education", n);
+    educationInput.innerHTML = `  
+      <hr/>
+      <a href="#" class="js-remove-education">
+        <i class="p-icon--close" style="float: right;"></i>
+      </a>
+      <label for="school-${n}">School</label>
+      <input list="school-${n}" name="school-${n}" type="text" class="js-school-${n}">
+      <datalist id="school-${n}">
+      </datalist>
+      <label for="degree-${n}">Degree</label>
+      <input list="degree-${n}" name="degree-${n}" type="text" class="js-degree-${n}">
+      <datalist id="degree-${n}">
+      </datalist>
+      <label for="discipline-${n}">Discipline</label>
+      <input list="discipline-${n}" name="discipline-${n}" type="text" class="js-dicipline-${n}">
+      <datalist id="discipline-${n}">
+      </datalist>
+      <div class="p-form--inline">
+        <div class="p-form__group">
+          <label class="p-form__label">Start date</label>
+          <div class="p-form__control u-clearfix">
+            <input placeholder="MM" id="start-month-${n}" name="start-month-${n}" type="text" class="p-form__control">
+          </div>
+          <div class="p-form__control u-clearfix">
+            <input placeholder="YYYY" id="start-year-${n}" name="start-year-${n}" type="text" class="p-form__control">
+          </div>
+        </div>
+      </div>
+      <div class="p-form--inline">
+        <div class="p-form__group">
+          <label class="p-form__label">End date</label>
+          <div class="p-form__control u-clearfix">
+            <input placeholder="MM" id="end-month-${n}" name="end-month-${n}" type="text" class="p-form__control">
+          </div>
+          <div class="p-form__control u-clearfix">
+            <input placeholder="YYYY" id="end-year-${n}" name="end-year-${n}" type="text" class="p-form__control">
+          </div>
+        </div>
+      </div>`
+    educationContainer.appendChild(educationInput);
+  };
+
+  schoolsList.addEventListener("input", debounce(async (e) => {
+    const datalistElement = document.querySelector("#school-0");
```

Comment:
This should be looking at the target otherwise the resulting list is based on first input